rca/scholarships/models.py

```diff
@@ -235,12 +245,24 @@ def get_context(self, request, *args, **kwargs):
                     for s in queryset
                 ]
 
+        # Create the link for the sticky CTA
+        interest_bar_link = reverse("scholarships:scholarship_enquiry_form")
+        if programme:
+            interest_bar_link += f"?{urlencode({'programme': programme.slug})}"
+
         context.update(
+            anchor_nav=self.anchor_nav(),
             filters={
-                "title": "Filter by",
+                "title": _("Filter by"),
                 "aria_label": "Filter results",
                 "items": filters,
             },
+            interest_bar={
+                "action": _("Express interest"),
+                "link": interest_bar_link,
+                "message": _("Hold an offer and want to apply for these Scholarships?"),
```

Comment:
Hold an offer? This doesn't make sense to me, but it's matching the designs so maybe check with RCA